# This contains the desired commands for the controller (mostly used for debugging)
float32 thrust_desired
float32 yaw_rate_desired
float32 roll_desired
float32 pitch_desired
